webrtc-stats.html

```diff
@@ -2695,8 +2695,9 @@ <h2>
               <dd>
                 <p>
                   The total number of samples that are concealed samples. A concealed sample is a
-                  sample that is based on data that was synthesized to conceal packet loss and does
-                  not represent incoming data.
+                  sample that is based on data that was synthesized to conceal packet loss or
+                  packets arriving late and does not represent incoming data. These samples are
+                  generated when we don't have the necessary non-concealed samples in the buffer.
```

Comment:
The total number of samples that conceal samples. A conceal sample is a sample that is based on data that was synthesized to **conceal samples from a missing packet during playout**. Concealment samples may be inserted when a packet is lost (reported in <a>packetsLost</a>) or the packet arrives to late/early to be played out (reported in <a>packetsDiscarded</a>).

Reply:
I dont quite remember what this means --- "does not represent incoming data" --> is this to disambiguate against datachannel?

Reply:
> These samples are generated when we don't have the necessary non-concealed samples in the buffer.

I dont think this is accurate, if playing out timestamp `t` and is missing packets, and the packets in the Jitter Buffer are already present at (t+200ms), would the application not add concealment samples??

Reply:
See ivoc's reply.